Makefile

```diff
@@ -87,7 +87,7 @@ manifests: ## Generate CustomResourceDefinition objects.
 generate: ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
 	go mod download k8s.io/code-generator
 	./hack/update-codegen.sh
-	go generate ./...
+$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
```

Comment:
controller gen is already used in ./hack/update-codegen.sh, no need to call it twice
jfyi, tabs are important in makefiles, so your added line actually is not invoked at all

Reply:
@mneverov I checked out the previous commit, added the boilerplate text to `./hack/update-codegen.sh` and ran `make generate`. It looks like it's enough to add the boilerplate license header to all the generated files. Why do we need to modify the `Makefile` at all?

If we want to make it like [here](https://github.com/kubernetes-sigs/kubebuilder/blob/master/docs/book/src/getting-started/testdata/project/Makefile#L51), does it mean that we want to get rid of `./hack/update-codegen.sh` and use `$(CONTROLLER_GEN)` directly?

Reply:
> `It looks like it's enough to add the boilerplate license header to all the generated files.`
that's true

> `Why do we need to modify the Makefile at all?`
we don't

my comment regarding adjusting makefile was about 1. making sure that it works; 2. if it doesn't use example from kubebuilder.

Since we already have `headerFile="..."` param in the update-codegen.sh no need to change anything there

Reply:
@mneverov I removed the unnecessary line from the makefile. Please, let me know if anything else needs to be done here.